modules/modify-rhsso-requests-limits.adoc

```diff
@@ -0,0 +1,30 @@
+// Module is included in the following assemblies:
+//
+// * installing-red-hat-openshift-gitops
+
+:_content-type: PROCEDURE
+[id="modifying-rhsso-resource-requests-limits_{context}"]
+= Modifying RHSSO resource requests/limits
+
+[role="_abstract"]
+By default, the RHSSO container is created with default resource requests and limits as shown below:
```

Comment:
Isn't this module a procedure? 

If so, consider rephrasing the abstract to: 

`By default, the RHSSO container is created with resource requests and limitations. You can change and manage the resource requests.`